dashboard/app/views/reports/search_for_teachers.html.haml

```diff
@@ -0,0 +1,65 @@
+- content_for(:head) do
+  = stylesheet_link_tag    'jquery.handsontable.full', media: 'all'
+  = javascript_include_tag 'jquery.handsontable.full'
+%h1 Search for Teachers
+
+%table
+  %tr
+    %td
+      Email Filter
+    %td
+      %input#emailFilter
+  %tr
+    %td
+      Address Filter
+    %td
+      %input#addressFilter
+
+%br/
+
+%center
+  %a#search
+    %button Search
+
+%br/
+
+#teacherTable
+
+:javascript
```

Comment:
you can still use handsontable if you want -- but to handle the inputs and navigation I think it would be simpler to use html forms than javascript (you're not using ajax requests or anything?), assuming the url prettifying is not necessary and you're ok with the url being `search_for_teachers?emailFilter=asd&addressFilter=sdf`.